lib/scoped/index.js

```diff
@@ -32,8 +33,13 @@ function scoped (api, type) {
     }
   }
 
-  api.on('change', require('../utils/handle-type-change').bind(null, emitter, type))
+  var state = {
+    storeObjectIds: map(scopedApi.findAll(), function (object) {
+      return object.id
+    })
```

Comment:
note that findAll() does not return an array, but a promise that resolves with an array